packages/@expo/config-plugins/src/android/Updates.ts

```diff
@@ -57,6 +72,19 @@ export const withUpdates: ConfigPlugin<{ expoUsername: string | null }> = (
   });
 };
 
+const withRuntimeVersionResource = createStringsXmlPlugin(applyRuntimeVersionFromConfig, 'withRuntimeVersionResource');
+
+function applyRuntimeVersionFromConfig(
+  config: Pick<ExpoConfigUpdates, 'sdkVersion' | 'runtimeVersion'>,
+  stringsJSON: ResourceXML
+): ResourceXML {
+  const runtimeVersion = getRuntimeVersionNullable(config, "android");
+  if (runtimeVersion) {
+    return setStringItem([buildResourceItem({ name: 'runtime_version', value: runtimeVersion })], stringsJSON);
+  }
+  return removeStringItem('app_name', stringsJSON);
```

Comment:
Doesn’t look like the right string. Be sure to write some unit tests for this

Reply:
Fixed and added unit tests